.github/workflows/build_and_test_feature.yml

```diff
@@ -256,6 +256,30 @@ jobs:
              yarn --cwd packages/react-component-library test:a11y
 
 
+     Test_design-tokens:
+       runs-on: ubuntu-latest
+       needs: [Build_icon_library, Lint_react-component-library]
+       steps:
+         - name: Git clone repository
+           uses: actions/checkout@v2
+
+         - name: Cache Node modules
+           uses: actions/cache@v2
+           with:
+             path: '**/node_modules'
+             key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+
+         - name: Attach workspace
+           uses: actions/download-artifact@v2
+           with:
+             name: dist
+
+         - name: Jest design-tokens
+           run: |
+             tar -xzf dist.tar.gz && mv distil packages/icon-library/dist && mv distdt packages/design-tokens/dist
```

Comment:
I think you can probably run the tests without needing the built versions of those other packages. So this line is maybe redundant?

Reply:
Thanks, have removed now